dask/dot.py

```diff
@@ -1,99 +1,74 @@
 from __future__ import absolute_import, division, print_function
 
-import networkx as nx
-from dask.core import istask, get_dependencies
+import os
 
+from graphviz import Digraph
 
-def make_hashable(x):
-    try:
-        hash(x)
-        assert len(str(x)) < 100
-        return x
-    except:
-        return hash(str(x))
-
+from .core import istask, get_dependencies, ishashable
 
-def lower(func):
-    while hasattr(func, 'func'):
-        func = func.func
-    return func
 
-def name(func):
+def label(func):
     try:
-        return lower(func).__name__
+        while hasattr(func, 'func'):
+            func = func.func
+        return func.__name__
     except AttributeError:
         return 'func'
 
 
-def to_networkx(d, data_attributes=None, function_attributes=None):
+def name(x):
+    try:
+        return str(hash(x))
+    except TypeError:
+        return str(hash(str(x)))
```

Comment:
Hashes make thiings harder to debug.  Thoughts on continuuing to return x if x is hashable?